changelog/8012.doc.rst

```diff
@@ -0,0 +1 @@
+Added an example `examples.plotting.add_countour_with_Spherical_screen` to the gallery to show that `sunpy.coordinates.screens.SphericalScreen` can be used even without reprojection
```

Comment:
`<unknown>:27: WARNING: py:obj reference target not found: examples.plotting.add_countour_with_Spherical_screen [ref.obj]`

This is reference is wrong.